xml/ssh_banners.xml

```diff
@@ -778,10 +778,11 @@ fingerprint SSH servers.
       <param pos="0" name="os.product" value="Windows"/>
    </fingerprint>
    
-   <fingerprint pattern="^Comware-(\d+\.?\d*\.?\d*)$">
+   <fingerprint pattern="^(?:SSH-[1|2]\.\d{1,2}-)?Comware-(\d+\.?\d*\.?\d*)$">
```

Comment:
Odd... do the real banners include double SSH- in them?  This file (for better or worse) is meant to be matched against everything after the SSH-<version>. IOW, if the full banner was:

```
SSH-2.0-OpenSSH_5.9p1 Debian-5ubuntu1.4
```

What recog would be matching against is:

```
OpenSSH_5.9p1 Debian-5ubuntu1.4
```